x-pack/test/functional/apps/monitoring/_get_lifecycle_methods.js

```diff
@@ -19,6 +20,9 @@ export const getLifecycleMethods = (getService, getPageObjects) => {
 
       await browser.setWindowSize(1600, 1000);
 
+      try {
```

Comment:
Maybe add a comment that indicates this is temporary / a hack:

```
// HACK: remove this once Elasticsearch index templates are updated
// https://github.com/elastic/elasticsearch/issues/37773
```

Also, we may want to avoid the try / catch and instead do a check if the index template exists, then only delete if so. That way, if deletion actually fails for reasons other than "doesn't exist", we'll get a reasonable error.

Other than that, this is fine with me.